client_endpoint.bal

```diff
@@ -64,18 +73,23 @@ public client class DataPlaneClient {
     # + containerId - ID of the container which contains the existing document
     # + document - A JSON document which will replace the existing document
     # + partitionKey - The specific value related to the partition key field of the container 
-    # + documentReplaceOptions - Optional. The `DocumentReplaceOptions` which can be used to add additional capabilities 
-    #                            to the request.
-    # + return - If successful, returns a `Document`. Else returns `error`.
-    remote function replaceDocument(string databaseId, string containerId, 
-                                    @tainted record {|string id; json...;|} document, 
-                                    int|float|decimal|string partitionKey, 
-                                    DocumentReplaceOptions? documentReplaceOptions = ()) 
-                                    returns @tainted Document|error {
+    # + documentReplaceOptions - The `DocumentReplaceOptions` which can be used to add additional capabilities to the 
+    #                            request
+    # + return - If successful, returns a `Document`. Else returns `Error`.
+    @display{
+        label: "Replace document"
+    } 
+    remote function replaceDocument(@display{label: "Database id"} string databaseId, 
```

Comment:
Shall we add a space after the display kewword?
```
@display {label: "Database id"} string databaseId
```

Reply:
Fixed. Thanks for the information